src/Core/AdminConsole/OrganizationFeatures/OrganizationUsers/ConfirmOrganizationUserCommand.cs

```diff
@@ -196,19 +196,19 @@ private async Task CheckPoliciesAsync(Guid organizationId, User user,
 
         if (_featureService.IsEnabled(FeatureFlagKeys.AutomaticConfirmUsers))
         {
-            var error = (await _automaticUserConfirmationPolicyEnforcementQuery.IsCompliantAsync(
+            var error = (await _automaticUserConfirmationPolicyEnforcementValidator.IsCompliantAsync(
```

Comment:
This could also use the bulk db query - this method is called in a loop. (Although it currently has looped db queries in it 😭 let's not make it worse if we can help)